<launch> 
<group>
		<node pkg="coop_localization" type="square" name="square1" output="screen" ns="/iris1">
			<param name="namespace" value="/iris1"/>
			<param name="use_sim_time"  value="true" />
            <param name="id" value="1" />
            <param name="height" value="2" />
		</node>
</group>
</launch>